wasmtime-runtime/signalhandlers/Trampolines.cpp

```diff
@@ -0,0 +1,37 @@
+#include <setjmp.h>
+
+#include "SignalHandlers.hpp"
+
+extern "C"
+int WasmtimeCallTrampoline(void *vmctx, void (*body)(void*, void*), void *args) {
+  jmp_buf buf;
+  void *prev;
```

Comment:
This should be volatile, `void *volatile prev`, to ensure that we see the correct value after the longjmp.